app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,24 @@
+<resources>
+    <string name="app_name">Snapdrop</string>
+
+    <string name="app_update">Atualização disponível</string>
+    <string name="app_update_summary">As atualizações contêm correções de falhas ou novas funcionalidades. A instalação é recomendada…</string>
+    <string name="app_update_install">Instalar</string>
+    <string name="app_update_show_details">Mais detalhes</string>
+    <string name="app_update_download_info">Baixando atualização</string>
+    <string name="app_update_download_text">Deve levar apenas alguns segundos…</string>
+    <string name="download_successful">Foi baixado com sucesso</string>
+    <string name="notification_channel_name">Download concluído</string>
+    <string name="title_activity_about">Sobre o Snapdrop para Android</string>
+    <string name="title_activity_about_short">Sobre o Snapdrop</string>
+    <string name="developer_notice" translatable="false">Snapdrop para Android por <a href="https://github.com/fm-sys">fm-sys</a></string>
```

Comment:
You are totally right, that this string should also be translated. Could you therefore please remove the `translatable="false"` flag from this string resource. Best both here and in `values/strings.xml`? 

Thanks for helping to translate this app!

Reply:
Hello fm-sys, 
I removed this section from both values. Thanks